lib/worker/index.js

```diff
@@ -15,6 +16,12 @@ const { CONCURRENT_WORKERS = 1 } = process.env
 const AxiosErrorEventDecorator = require('../models/axios-error-event-decorator')
 const SentryScopeProxy = require('../models/sentry-scope-proxy')
 
+function measureElapsedTime (startTime, tags) {
+  const endTime = new Date()
+  var timeDiff = endTime - startTime
+  statsd.histogram('job_duration', timeDiff, tags)
```

Comment:
These histograms will give us several nice pieces of data for completed and failed jobs. Including counts, timing, and percentiles.

Reply:
Not necessary, but I find it very helpful to indicate the time unit in the metric name. Something like `job_duration_sec` or `job_duration_ms` to make dashboarding easier.

Reply:
Yeah, I'm not a big fan of the Hungarian notation for the statsd names. I'll change the datatype in datadog once we start collecting metrics.

Reply:
I understand that. The default for datadog is `ms` if I recall, maybe if we just make sure that we're reporting this in ms (I think we are) then that's good!

Reply:
> I understand that. The default for datadog is ms if I recall, maybe if we just make sure that we're reporting this in ms (I think we are) then that's good!

Ahh yeah! We are! ms is the default:

```javascript
var a = new Date();
undefined
var b = new Date();
undefined
b - a
811
```